modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -198,7 +198,7 @@ public void testAddObjectRelationship() throws Exception {
 				ObjectRelationshipConstants.TYPE_MANY_TO_MANY +
 					" does not allow a parameter object field ID",
 			() -> _objectRelationshipLocalService.addObjectRelationship(
-				TestPropsValues.getUserId(),
+				null, TestPropsValues.getUserId(),
```

Comment:
Could you move these changes to [this commit](https://github.com/liferay-objects/liferay-portal/pull/2788/commits/46df3bdf012bffe02d75018df9481b5757ab14a1)? 
You can follow [this example](https://github.com/liferay-objects/liferay-portal/pull/2239/commits/f2019700d6aae4a8436c35b03979834b68ef58b0) from my PR